src/main/java/homeWork23/ui/LoginPage.java

```diff
@@ -0,0 +1,25 @@
+package homeWork23.ui;
+
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class LoginPage {
+
+    private final SelenideElement userNameInput = $("input[data-placeholder='Username']");
+    private final SelenideElement userPasswordInput = $("input[data-placeholder='Password']");
+    private final SelenideElement loginBtn = $("button.mat-raised-button.mat-primary");
+    private final SelenideElement shownUserName = $x("//span[@class='mat-button-wrapper' " +
+            "and mat-icon[@role='img'][@class='mat-icon notranslate material-icons mat-icon-no-color'][1] " +
+            "and mat-icon[@role='img'][@class='mat-icon notranslate material-icons mat-icon-no-color'][2]]");
```

Comment:
Can this locator be simplified?
E.g. ".mat-focus-indicator.mat-menu-trigger.mat-button.mat-button-base.ng-star-inserted"
(just a suggestion)

Reply:
i had a certain issues with that (somehow user name is cathced only if exact path to that name is specified), but xpath was simplified